routes/api.coffee

```diff
@@ -1,3 +1,7 @@
 module.exports = (app, api, db) ->
   app.get '/api/v1/campaigns', api.v1.getCampaigns
   app.get '/api/v1/campaigns/:id', api.v1.getCampaign
+  app.get '/api/v1/candidates', api.v1.getCandidates
+  app.get '/api/v1/candidates/:id', api.v1.getCandidate
+  app.get '/api/v1/search', api.v1.search
```

Comment:
This should be `/api/v1/candidates/search` and named `api.v1.candidatesSearch` accordingly. You'll probably need to declare it before the get candidates handler to ensure it runs.